<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListDemo</class>
 <widget class="QMainWindow" name="ListDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>783</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>783</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>783</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ListDemo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ListDemo{
background-color: rgb(255, 255, 255);
	border-image: url(:/backpage_common.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_num">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px groove gray;;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
    </property>
    <property name="placeholderText">
     <string>序号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_operate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/arrow_down.png);
}


QComboBox#cbb_operate{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_operate:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>请选择操作</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>创建</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>插入</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>删除</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>倒置</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>791</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,128);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_ok{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_ok:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_ok:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_ok:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_text">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px groove gray;;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
    </property>
    <property name="placeholderText">
     <string>内容</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>791</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,128);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Debug">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_ok{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_ok:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_ok:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_ok:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择链表类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_insertType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/arrow_down.png);
}


QComboBox#cbb_insertType{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_insertType:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>请选择插入类型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按序号前插</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按序号后插</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按值前插</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按值后插</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按值全部前插</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按值全部后插</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbb_deleteType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/arrow_down.png);
}


QComboBox#cbb_deleteType{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_deleteType:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>请选择删除类型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按序号删除</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按值删除</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按值全部删除</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rbt_seq">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>顺序表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbt_link">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>链表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
